metricbeat/module/kvm/dommemstat/dommemstat.go

```diff
@@ -149,8 +151,13 @@ func (m *MetricSet) Fetch(report mb.ReporterV2) {
 					},
 				},
 			})
+			if !reported {
+				break
```

Comment:
this will go to the next domain, you probably want to return here?

Reply:
Ah yah, think you're right.